samples/snippets/storage_download_into_memory.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+# [START storage_download_file]
+from google.cloud import storage
+
+
+def download_blob_into_memory(bucket_name, source_blob_name):
+    """Downloads a blob from the bucket."""
```

Comment:
Thanks for adding this! Could we update the description to """Downloads a blob into memory.""" and change the parameter name to `blob_name`? The word source might be a bit confusing?

Reply:
done!